quinn-proto/src/connection.rs

```diff
@@ -545,18 +552,18 @@ impl Connection {
         false
     }
 
-    fn set_key_discard_timer(&mut self, now: u64) {
+    fn set_key_discard_timer(&mut self, now: Instant) {
         let time = if self.spaces[SpaceId::Handshake as usize].crypto.is_some() {
             now + self.pto() * 3
         } else if let Some(ref time) = self.prev_crypto.as_ref().and_then(|x| x.update_ack_time) {
-            time + self.pto() * 3
+            *time + self.pto() * 3
```

Comment:
Style nit: get rid of the `ref` above rather than dereffing?

Reply:
Fixed.